web/src/containers/AuthChecker.js

```diff
@@ -13,7 +13,7 @@ class AuthChecker extends Component {
     const { authInit, children } = this.props;
 
     if (!authInit) return null;
-    return <div>{children}</div>;
+    return <Fragment>{children}</Fragment>;
```

Comment:
the wrapping `div` here was unnecessary (and more importantly, was throwing off the full-width-ness of the sidebar)